### PR TITLE
Created Toggle Button for Disabling Presence

### PR DESCRIPTION
**Pull request by ThomasAunvik**
Added Feature #11

### CODE REVIEW

File: discord-rpc-vs/Config/Configuration.cs [hunk 1]
```diff
@@ -10,6 +10,12 @@ namespace discord_rpc_vs.Config
 {
     internal class Configuration
     {
+
+        /// <summary>
+        ///     Dictates whether or not to enable presence
+        /// </summary>
+        public bool PresenceDisabled { get; set; }
```
Comment (Swan) [inline discord-rpc-vs/Config/Configuration.cs]: This should be renamed to PresenceEnabled, just for consistency's sake, and all of its corresponding uses should be flipped the opposite way.

File: discord-rpc-vs/Config/Configuration.cs [hunk 2]
```diff
@@ -42,6 +48,7 @@ internal static Configuration Deserialize()
             if (!File.Exists(path))
             {
                 var config = new Configuration();
+                config.PresenceDisabled = true;
```
Comment (Swan) [inline discord-rpc-vs/Config/Configuration.cs]: The presence should be enabled by default

File: discord-rpc-vs/DiscordRPCVSPackage.cs [hunk 3]
```diff
@@ -171,7 +177,7 @@ private void OnWindowSwitch(Window windowActivated, Window lastWindow)
             else if (Config.DisplayTimestamp && !Config.ResetTimestamp)
                 DiscordController.presence.startTimestamp = InitialTimestamp;
 
-            DiscordRPC.UpdatePresence(ref DiscordController.presence);
+            if(!Config.PresenceDisabled) DiscordRPC.UpdatePresence(ref DiscordController.presence);
```
Comment (Swan) [inline discord-rpc-vs/DiscordRPCVSPackage.cs]: Put the method call on a new line.

File: discord-rpc-vs/TogglePresence.cs [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+            if (sender is OleMenuCommand menuCommand)
+            {
+                menuCommand.Checked = DiscordRPCVSPackage.Config.PresenceDisabled;
+                CheckIfShouldDisable();
+
+                if (!DiscordRPCVSPackage.Config.PresenceDisabled) EnableRPC();
+            }
+        }
+
+        private void CheckIfShouldDisable()
```
Comment (Swan) [inline discord-rpc-vs/TogglePresence.cs]: Add some XML documentation, even though it's pretty obvious what it does.

File: discord-rpc-vs/TogglePresence.cs [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+        {
+            if (DiscordRPCVSPackage.Config.PresenceDisabled)
+            {
+                DiscordRPC.Shutdown();
+            }
+        }
+
+        private void EnableRPC()
```
Comment (Swan) [inline discord-rpc-vs/TogglePresence.cs]: Same thing. Add some documentation just for consistency.

File: discord-rpc-vs/Config/Configuration.cs (hunk 2 above)
Comment (ThomasAunvik) [reply discord-rpc-vs/Config/Configuration.cs]: I first had `PresenceEnabled` but then i noticed the default was not on true, cause the file did exist, which keeps it `false` when you already have a save file.

**Comment by ThomasAunvik** [comment]
Requesting re-review @Swan

File: discord-rpc-vs/Config/Configuration.cs [hunk 6]
```diff
@@ -14,7 +14,7 @@ internal class Configuration
         /// <summary>
         ///     Dictates whether or not to enable presence
         /// </summary>
-        public bool PresenceDisabled { get; set; }
+        public bool PresenceEnabled { get; set; }
```
Comment (Swan) [inline discord-rpc-vs/Config/Configuration.cs]: Setting this to true (and all the other ones), should allow it to be defaulted correctly, so if you could default this to true along with all other properties then we should be good to go.

Comment (ThomasAunvik) [reply discord-rpc-vs/Config/Configuration.cs]: The latest commit should do the trick, if that is what you meant.

**Comment by Swan** [comment]
Good stuff

**Pull request merged**